.github/workflows/maven.yml

```diff
@@ -0,0 +1,13 @@
+name: Verify
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Verify
+    uses: jetty/.github/.github/workflows/maven-ci.yml@main
+    with:
+      jdk-matrix: '[ "11", "17', "21" ]'
+      os-matrix: '[ "ubuntu-latest", "windows-latest", "macOS-latest" ]'
```

Comment:
only `ubuntu-latest` here

Reply:
why? afaik some people are using those os and as this library is doing some files manipulation it may worth to test on different os

Reply:
of the 6,100 uses of jetty-test-helper on github ...

Eliminate Jetty itself and all of the various forks of Jetty and you are left with ...

12 projects that mention it in their poms.
All of those only mention it in dependency / excludes

Eg: https://github.com/apache/drill/blob/master/exec/java-exec/pom.xml#L190

Reply:
Only ubuntu-latest here.

Reply:
This configuration would also not be relevant to PRs, only to pushes on the `main` branch.

Also of note about the hosted runners:
 * `macOS-latest` is in beta and only uses `macos-13` (the majority of our users are still on `macos-12`, which doesn't exist as a runner)
 * `windows-latest` is for `windows-2022` (a Windows Server instance) which will not tell us anything useful about the build if it fails, it's considered to be in beta for java builds. (the `windows-2019` runner is equally as useless for java builds)

Reply:
AND those runners use about 8x of our GH action quota for no good reason.
Drop them, there's zero reason to use them for Java projects.